lua/ide/components/bufferlist/component.lua

```diff
@@ -0,0 +1,247 @@
+local base = require("ide.panels.component")
+local libwin = require("ide.lib.win")
+local logger = require("ide.logger.logger")
+local icon_set = require("ide.icons").global_icon_set
+local commands = require("ide.components.bufferlist.commands")
+
+local BufferListComponent = {}
+
+local config_prototype = {
```

Comment:
Should this be in libbuf? 
https://github.com/ldelossa/nvim-ide/blob/main/lua/ide/lib/buf.lua

Reply:
Maybe I'm confused or you commented on the wrong line by mistake, this is the component config, why would this go in libbuf?

Reply:
yup this is an error